packages/next/lib/typescript/writeConfigurationDefaults.ts

```diff
@@ -183,7 +183,7 @@ export async function writeConfigurationDefaults(
     chalk.green(
       `We detected TypeScript in your project and reconfigured your ${chalk.bold(
         'tsconfig.json'
-      )} file for you.`
+      )} file for you. Strict-mode is set to ${chalk.bold('false')} by default.`
```

Comment:
Hi! Thanks for adding this, but a quick question: If someone doesn't know exactly what this means, is this helpful? This should most likely link to an article CA about the implications of strict mode and how it's intended to be enabled.

Reply:
At least it is helpful for people who would expect strict mode (which is the default of `tsc --init`), so it's definitely a win.

But yeah, people starting their first TypeScript project with this probably might read over it and not assume that `strictNullChecks` is off or what the implications of that are.